config/karma.config.js

```diff
@@ -23,6 +23,7 @@ const karmaConfig = {
   browsers : ['PhantomJS'],
   webpack  : {
     devtool : 'cheap-module-source-map',
+    entry: './tests/test-bundler.js',
```

Comment:
Should this be `./${project.dir_test}/test-bundler.js` like on [line 11](https://github.com/davezuko/react-redux-starter-kit/pull/1189/files#diff-c3ce07ad1a971575599220c4a08f29f0R11) of this file